pom.xml

```diff
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>nexus-plugins</artifactId>
-    <version>3.19.0-01</version>
+    <version>3.20.0-SNAPSHOT</version>
   </parent>
```

Comment:
why you did not use the latest version?  The latest is 3.21 ?

Reply:
It's not released and it should be a separate ticket